latest/index.html

```diff
@@ -218,6 +218,7 @@ <h4>This Draft</h4>
           <p>Collection of changes which make the API conform to the REST principles.</p>
           <ul>
             <li>Remove support for flyouts;</li>
+            <li>Remove support for supplying a single type in a reconciliation query without enclosing it in an array;</li>
```

Comment:
Yes, I have added a line about this here - would you want to see that elsewhere? Or written up differently?

Reply:
"Conform to REST principles" doesn't really draw the eye in the same was as "Incompatible changes" or some such, so I'd recommend a separate section. Ideally, it should be easy for a developer of a client or service to understand the implications of the change as described.